extensions/jackson/runtime/src/main/java/io/quarkus/jackson/runtime/JacksonBuildTimeConfig.java

```diff
@@ -43,4 +45,10 @@
      */
     @ConfigItem(defaultValue = "UTC")
     public Optional<ZoneId> timezone;
+
+    /**
+     * Define which properties of Java Beans are to be included in serialization.
+     */
+    @ConfigItem
+    public Optional<JsonInclude.Include> serializationInclusion;
```

Comment:
I haven't used this Jackson property, but does it allow to use multiple values?

Reply:
I had the same question and I checked and it doesn't.